lib/pedersen/index.js

```diff
@@ -68,12 +68,12 @@ function commit(blindFactor, value, generator) {
  *  @summary Serializes a pedersen commitment.
  *  @return {Array} 33-bytes serialized pedersen commitment.
```

Comment:
64

Reply:
How it is that input has 64 and output 33?

Reply:
The commitment serialization takes a 64 byte raw pedersen commitment as input and return a 33 byte serialized commit as output.